test/common/lcov.py

```diff
@@ -391,10 +391,14 @@ def get_review_comments(diff_info_file):
     cur_src = None
     start_line = None
     cur_line = None
+    cur_text = ""
 
     def flush_cur_comment():
         nonlocal comments
-        if cur_src:
+        # Don't complain about added regions that contain only
+        # punctuation, or nothing but "else".
+        if cur_src and re.search('[a-zA-Z0-9]', cur_text.replace("else", "")):
+            print("C", cur_text)
```

Comment:
Is this print a debugging leftover, and I'm also wondering if this should go into a separate PR?

Reply:
This commit is already in main, I'll remove it here.

Reply:
(But yes, debug leftover)